site/src/components/poll-worker-info/poll-worker-info.tsx

```diff
@@ -38,7 +37,7 @@ export class PollWorkerInfo {
             ? jurisdictionId
                ? (<jurisdiction-info jurisdictionId={jurisdictionId} />)
                : (<state-info state={state} stateId={stateId} />)
-            : (<h3>Enter state, county, city</h3>) }
+            : <stencil-route-redirect url="/search" /> }
```

Comment:
@nexussays I don't see this work locally - the docs match so kind of confused

Reply:
Hmm. I've not used that component before so don't have any ready answers. Maybe use ` @Prop() history: RouterHistory;` instead? https://github.com/ionic-team/stencil-router/wiki/Create-a-redirect#responding-to-a-user-action

Reply:
That did it - now redirecting when it can't find a state or jurisdiction